Name: Prevent Metrix Log Uploads
Version: 1.4.1-1
Author: Northguard
Description: Metrix collects personal details like device ID and carrier. This patch prevents log uploads.  (MIT license)

--- .orig/etc/hosts
+++ /etc/hosts
@@ -1,1 +1,2 @@
+127.0.0.1 metrix.webosroundup.com
 127.0.0.1	localhost.localdomain		localhost
